test/controllers/analytics.js

```diff
@@ -1,26 +1,33 @@
 const request = require('supertest');
 const chai = require('chai');
 
-const server = require('../../app');
+const User = require('../../api/models/User');
+
 const helper = require('../helper');
 
 //Parent block
-describe.skip('analytics API calls', function () {
+describe('analytics API calls', function () {
+  let server;
   let user;
 
   before(async function () {
-    await helper.deleteMochaUser();
-    user = await helper.prepareUser(server, {
-      role: 'user',
-      email: helper.userData.email,
-    });
+    helper.prepareNodemailerMock(); //enable mockery and replace nodemailer with nodemailerMock
+    server = require('../../app'); //register mocks before require the original dependency
   });
 
   after(async function () {
-    await helper.deleteMochaUser();
+    helper.prepareNodemailerMock(true);
```

Comment:
As far as I know we never need a real instance of Nodemailer during our tests, so maybe we could mock it only once in a root `before` hook: https://mochajs.org/#root-hook-plugins

Reply:
yes, we didn't need it. I will generate the root-hook file and try it.